install_sedumi.m

```diff
@@ -175,6 +175,10 @@ function install_sedumi( varargin )
       % whereas calling '-O' would result in gcc '-O1'.
       flags{end+1} = '-O2';
       flags{end+1} = '-DOCTAVE';
+      flags{end+1} = '-Werror';
+      flags{end+1} = '-Wall';
+      flags{end+1} = '-Wno-unused-variable';
+      flags{end+1} = '-Wno-unused-but-set-variable';
```

Comment:
Obviously not supported by clang (macOS) https://travis-ci.org/github/sqlp/sedumi/jobs/718183038

Is it too difficult to remove the unused variables as well?  Then those flags are not needed.